src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server;
+
+import io.swagger.annotations.*;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.inject.Inject;
+import java.util.UUID;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + NetworkModificationApi.API_VERSION + "/")
+@Api(tags = "network-modification-server")
+@ComponentScan(basePackageClasses = NetworkModificationService.class)
+public class NetworkModificationController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NetworkModificationController.class);
+
+    @Inject
+    private NetworkModificationService networkModificationService;
+
+    @PutMapping(value = "/switch/{networkUuid}/{switchId}")
```

Comment:
To follow classical REST API convention it should be:
/networks/{networkUuid}/switches/{switchId}

Reply:
Modifications done